taichi/backends/vulkan/runtime.cpp

```diff
@@ -293,15 +286,14 @@ size_t CompiledTaichiKernel::num_pipelines() const {
   return pipelines_.size();
 }
 
-DeviceAllocation *CompiledTaichiKernel::ctx_buffer() const {
-  return ctx_buffer_.get();
+size_t CompiledTaichiKernel::get_ctx_buffer_size() const {
+  return ctx_buffer_size_;
 }
 
-DeviceAllocation *CompiledTaichiKernel::ctx_buffer_host() const {
-  return ctx_buffer_host_.get();
-}
-
-void CompiledTaichiKernel::command_list(CommandList *cmdlist) const {
+void CompiledTaichiKernel::command_list(
```

Comment:
Out of curiosity, what does the function name `command_list` mean here, `run_command_list`?

Reply:
This means record command list

Reply:
nit: Can we just rename it to `record_command_list` then :-) ?